2023-12-25 04:30:45,612 INFO    StreamThr :28200 [internal.py:wandb_internal():86] W&B internal server running at pid: 28200, started at: 2023-12-25 04:30:45.611713
2023-12-25 04:30:45,613 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: status
2023-12-25 04:30:45,620 INFO    WriterThread:28200 [datastore.py:open_for_write():85] open: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\run-dvv52nzy.wandb
2023-12-25 04:30:45,621 DEBUG   SenderThread:28200 [sender.py:send():382] send: header
2023-12-25 04:30:45,621 DEBUG   SenderThread:28200 [sender.py:send():382] send: run
2023-12-25 04:30:45,928 INFO    SenderThread:28200 [dir_watcher.py:__init__():211] watching files in: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files
2023-12-25 04:30:45,928 INFO    SenderThread:28200 [sender.py:_start_run_threads():1136] run started: dvv52nzy with start time 1703496645.613709
2023-12-25 04:30:45,931 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: check_version
2023-12-25 04:30:45,932 DEBUG   SenderThread:28200 [sender.py:send_request():409] send_request: check_version
2023-12-25 04:30:46,118 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: run_start
2023-12-25 04:30:47,409 DEBUG   HandlerThread:28200 [system_info.py:__init__():32] System info init
2023-12-25 04:30:47,409 DEBUG   HandlerThread:28200 [system_info.py:__init__():47] System info init done
2023-12-25 04:30:47,409 INFO    HandlerThread:28200 [system_monitor.py:start():194] Starting system monitor
2023-12-25 04:30:47,409 INFO    SystemMonitor:28200 [system_monitor.py:_start():158] Starting system asset monitoring threads
2023-12-25 04:30:47,410 INFO    HandlerThread:28200 [system_monitor.py:probe():214] Collecting system info
2023-12-25 04:30:47,415 INFO    SystemMonitor:28200 [interfaces.py:start():190] Started cpu monitoring
2023-12-25 04:30:47,415 INFO    SystemMonitor:28200 [interfaces.py:start():190] Started disk monitoring
2023-12-25 04:30:47,416 INFO    SystemMonitor:28200 [interfaces.py:start():190] Started gpu monitoring
2023-12-25 04:30:47,417 INFO    SystemMonitor:28200 [interfaces.py:start():190] Started memory monitoring
2023-12-25 04:30:47,425 INFO    SystemMonitor:28200 [interfaces.py:start():190] Started network monitoring
2023-12-25 04:30:47,436 DEBUG   HandlerThread:28200 [system_info.py:probe():196] Probing system
2023-12-25 04:30:47,437 DEBUG   HandlerThread:28200 [gitlib.py:_init_repo():56] git repository is invalid
2023-12-25 04:30:47,437 DEBUG   HandlerThread:28200 [system_info.py:probe():244] Probing system done
2023-12-25 04:30:47,437 DEBUG   HandlerThread:28200 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.9.18', 'heartbeatAt': '2023-12-25T09:30:47.436750', 'startedAt': '2023-12-25T09:30:45.602273', 'docker': None, 'cuda': None, 'args': ('--yt_links_filename', 'data/temp.txt', '--output_path', 'TEMP'), 'state': 'running', 'program': 'Q:\\AD project\\transcripts_main.py', 'codePathLocal': 'transcripts_main.py', 'codePath': 'transcripts_main.py', 'host': 'LAPTOP-QF1FUNPH', 'username': 'astar', 'executable': 'C:\\Users\\astar\\anaconda3\\envs\\tf-gpu\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 1901.0, 'min': 0.0, 'max': 1901.0}, 'cpu_freq_per_core': [{'current': 1901.0, 'min': 0.0, 'max': 1901.0}], 'disk': {'/': {'total': 48.827144622802734, 'used': 43.55003356933594}}, 'gpu': 'NVIDIA GeForce RTX 3050 Ti Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Ti Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 15.355525970458984}}
2023-12-25 04:30:47,437 INFO    HandlerThread:28200 [system_monitor.py:probe():224] Finished collecting system info
2023-12-25 04:30:47,437 INFO    HandlerThread:28200 [system_monitor.py:probe():227] Publishing system info
2023-12-25 04:30:47,437 DEBUG   HandlerThread:28200 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2023-12-25 04:30:47,437 DEBUG   HandlerThread:28200 [system_info.py:_save_pip():68] Saving pip packages done
2023-12-25 04:30:47,438 DEBUG   HandlerThread:28200 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2023-12-25 04:30:47,442 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-12-25 04:30:47,450 ERROR   HandlerThread:28200 [system_info.py:_save_conda():86] Error saving conda packages: [WinError 2] The system cannot find the file specified
Traceback (most recent call last):
  File "C:\Users\astar\anaconda3\envs\tf-gpu\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 82, in _save_conda
    subprocess.call(
  File "C:\Users\astar\anaconda3\envs\tf-gpu\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\astar\anaconda3\envs\tf-gpu\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\astar\anaconda3\envs\tf-gpu\lib\subprocess.py", line 1436, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
2023-12-25 04:30:47,454 DEBUG   HandlerThread:28200 [system_info.py:_save_conda():87] Saving conda packages done
2023-12-25 04:30:47,454 INFO    HandlerThread:28200 [system_monitor.py:probe():229] Finished publishing system info
2023-12-25 04:30:47,462 DEBUG   SenderThread:28200 [sender.py:send():382] send: files
2023-12-25 04:30:47,462 INFO    SenderThread:28200 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2023-12-25 04:30:47,469 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: stop_status
2023-12-25 04:30:47,470 DEBUG   SenderThread:28200 [sender.py:send_request():409] send_request: stop_status
2023-12-25 04:30:47,471 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: internal_messages
2023-12-25 04:30:47,653 DEBUG   SenderThread:28200 [sender.py:send():382] send: telemetry
2023-12-25 04:30:47,946 INFO    Thread-16 :28200 [dir_watcher.py:_on_file_created():271] file/dir created: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files\wandb-metadata.json
2023-12-25 04:30:47,946 INFO    Thread-16 :28200 [dir_watcher.py:_on_file_created():271] file/dir created: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files\requirements.txt
2023-12-25 04:30:47,946 INFO    Thread-16 :28200 [dir_watcher.py:_on_file_created():271] file/dir created: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files\conda-environment.yaml
2023-12-25 04:30:48,097 INFO    wandb-upload_0:28200 [upload_job.py:push():131] Uploaded file C:\Users\astar\AppData\Local\Temp\tmpq43kc9q1wandb\kg5wh3wh-wandb-metadata.json
2023-12-25 04:30:49,937 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:49,938 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:50,687 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: status_report
2023-12-25 04:30:52,445 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:52,446 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:54,963 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:54,964 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:55,730 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: status_report
2023-12-25 04:30:57,527 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:30:57,527 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:31:00,071 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:31:00,072 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-12-25 04:31:01,046 INFO    Thread-16 :28200 [dir_watcher.py:_on_file_created():271] file/dir created: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files\output.log
2023-12-25 04:31:01,185 DEBUG   HandlerThread:28200 [handler.py:handle_request():146] handle_request: status_report
2023-12-25 04:31:02,091 ERROR   gpu       :28200 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-12-25 04:31:02,766 WARNING StreamThr :28200 [internal.py:is_dead():414] Internal process exiting, parent pid 6084 disappeared
2023-12-25 04:31:02,766 ERROR   StreamThr :28200 [internal.py:wandb_internal():152] Internal process shutdown.
2023-12-25 04:31:03,061 INFO    Thread-16 :28200 [dir_watcher.py:_on_file_modified():288] file/dir modified: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\files\output.log
2023-12-25 04:31:03,201 INFO    SenderThread:28200 [sender.py:finish():1548] shutting down sender
2023-12-25 04:31:03,201 INFO    HandlerThread:28200 [handler.py:finish():866] shutting down handler
2023-12-25 04:31:03,201 INFO    WriterThread:28200 [datastore.py:close():294] close: Q:\AD project\wandb\run-20231225_043045-dvv52nzy\run-dvv52nzy.wandb
2023-12-25 04:31:03,201 INFO    HandlerThread:28200 [system_monitor.py:finish():203] Stopping system monitor
2023-12-25 04:31:03,201 DEBUG   SystemMonitor:28200 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2023-12-25 04:31:03,201 INFO    HandlerThread:28200 [interfaces.py:finish():202] Joined cpu monitor
2023-12-25 04:31:03,202 DEBUG   SystemMonitor:28200 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2023-12-25 04:31:03,202 INFO    HandlerThread:28200 [interfaces.py:finish():202] Joined disk monitor
2023-12-25 04:31:03,202 DEBUG   SystemMonitor:28200 [system_monitor.py:_start():183] Publishing last batch of metrics
2023-12-25 04:31:03,693 INFO    HandlerThread:28200 [interfaces.py:finish():202] Joined gpu monitor
2023-12-25 04:31:03,693 INFO    HandlerThread:28200 [interfaces.py:finish():202] Joined memory monitor
2023-12-25 04:31:03,693 INFO    HandlerThread:28200 [interfaces.py:finish():202] Joined network monitor
2023-12-25 04:31:04,202 INFO    SenderThread:28200 [dir_watcher.py:finish():358] shutting down directory watcher
2023-12-25 04:31:04,280 INFO    MainThread:28200 [internal.py:handle_exit():76] Internal process exited
